editorial: This sentence was hard to parse for me (recovery) (#3383)
quicwg/base-drafts: draft-ietf-quic-recovery.md

largest acknowledged packet.

An RTT sample MUST NOT be generated on receiving an ACK frame that does not

> newly acknowledge at least one ack-eliciting packet.  A peer does usally not

Reviewer: s/usally/usually/

Also, I'd place "usually" before does:

> A peer usually does not send [...]

Reviewer 2: "does not usually" would also be fine.  But not in the middle.

Author: done

send an ACK frame when only non-ack-eliciting packets are received. Therefore

> an ACK frame that contains only acknowledge for non-ack-eliciting packets is

Reviewer 2: ```suggestion
an ACK frame that only contains acknowledgements for non-ack-eliciting packets
```
May require a rewrap

Author: thanks. done!

> an ACK frame that only contains acknowledgements for non-ack-eliciting packets

Reviewer 3: ```suggestion
an ACK frame that contains acknowledgements for only non-ack-eliciting packets
```

Reviewer 2: Technically, it could still acknowledge other packets as well -- the key is which packets are *newly* acknowledged, isn't it?

> can subsequently include an arbitrarily large Ack Delay value.  Ignoring

Reviewer 4: ```suggestion
could include an arbitrarily large Ack Delay value.  Ignoring
```
